tests/resources/aria-at-harness.mjs

```diff
@@ -715,15 +755,10 @@ function showUserError() {
   }
 }
 
-function reportResults(testResults, status) {
+function appendJSONResults(data) {
```

Comment:
When testing in the prototype, results are not getting put into a script tag -- which breaks the prototype runner.  I think it this function is not being called, but haven't dug in. You can test if you want by running a server from the /tests/ directory like (`python -m SimpleHTTPServer `) without the show results = false -- then complete the test, and see that the script tage doesn't appear.